docs/src/main/asciidoc/logging.adoc

```diff
@@ -328,17 +330,17 @@ The logging format string supports the following symbols:
 
 |%%|`%`|Renders a simple `%` character.
 |%c|Category|Renders the category name.
-|%C|Source class|Renders the source class name.footnote:calc[Format sequences which examine caller information may affect performance]
+|%C|Source class|Renders the source class name.footnoteref:[calc, Format sequences which examine caller information may affect performance]
```

Comment:
Unfortunately, this won't fly because we have a warning with recent asciidoc versions and we have to error out on warnings as some might be real problems we need to fix:

```
2024-04-15T14:00:25.5928425Z [INFO] asciidoctor: WARN: found deprecated footnoteref macro: footnoteref:[calc, Format sequences which examine caller information may affect performance]; use footnote macro with target instead
2024-04-15T14:00:25.5934178Z [INFO] asciidoctor: WARN: found deprecated footnoteref macro: footnoteref:[calc]; use footnote macro with target instead
2024-04-15T14:00:25.5941585Z [INFO] asciidoctor: WARN: found deprecated footnoteref macro: footnoteref:[calc]; use footnote macro with target instead
2024-04-15T14:00:25.5944198Z [INFO] asciidoctor: WARN: found deprecated footnoteref macro: footnoteref:[calc]; use footnote macro with target instead
2024-04-15T14:00:25.5948018Z [INFO] asciidoctor: WARN: found deprecated footnoteref macro: footnoteref:[calc]; use footnote macro with target instead
2024-04-15T14:00:25.7144867Z [INFO] Converted /home/runner/work/quarkus/quarkus/docs/target/asciidoc/sources/logging.adoc
2024-04-15T14:00:25.7149754Z [ERROR] asciidoctor: WARN: found deprecated footnoteref macro: footnoteref:[calc, Format sequences which examine caller information may affect performance]; use footnote macro with target instead
2024-04-15T14:00:25.7151515Z [ERROR] asciidoctor: WARN: found deprecated footnoteref macro: footnoteref:[calc]; use footnote macro with target instead
2024-04-15T14:00:25.7152773Z [ERROR] asciidoctor: WARN: found deprecated footnoteref macro: footnoteref:[calc]; use footnote macro with target instead
2024-04-15T14:00:25.7154005Z [ERROR] asciidoctor: WARN: found deprecated footnoteref macro: footnoteref:[calc]; use footnote macro with target instead
2024-04-15T14:00:25.7155229Z [ERROR] asciidoctor: WARN: found deprecated footnoteref macro: footnoteref:[calc]; use footnote macro with target instead
```

Please keep them as they are and I will rewrite them in the downstream Java toolchain.

Reply:
See https://github.com/quarkusio/quarkus/pull/40082 .

You should be able to revert these specific changes safely.

Reply:
You are golden @gsmet ! Thank you.
1) I will revert the changes made to these footnotes.
2) Regarding the pluses used to align text to the steps, thus create some more space for the very tiny heading that is below - this is very rare, I have no intentions to do this anywhere else. It is simple asciidoc and it works in Upstream, solves that "space" problem, and overall rising the level of readability - even thought I know this is tweak that makes things more shiny, but, potentially, less resilient. If you recommend to revert these changes too, I will do it, but from my point of view it will do no harm.

Reply:
@gsmet Reverted! Repushed with fresh rebase.